src/styles/main.scss

```diff
@@ -1,3 +1,20 @@
+@import "utils/variables";
+@import "utils/functions";
+@import "reset";
+
+@import "blocks/header";
+@import "blocks/nav";
+@import "blocks/content";
+@import "blocks/product-card";
+@import "blocks/stars";
+
 body {
   margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+}
+
+.logo {
```

Comment:
Tip: For the future, this .logo class should be moved to your header.scss file as it's part of the page header. Remember about that :)

Reply:
I moved logo`to header block.